install_config/aggregate_logging.adoc

```diff
@@ -3299,3 +3299,118 @@ link:https://github.com/Scribery/tlog[Tlog].
 |Scrubbed invalid output characters as bytes.
 |===
 endif::openshift-origin[]
+
+== Manual Elasticsearch rollouts
+
+The latest release of the Aggregated Logging stack has updated the Elasticsearch
+Deployment Config object so that it no longer has a Config Change Trigger meaning
+any changes to the DC will not result in an automatic rollout. This was to prevent
+unintended restarts happening in the ES cluster which could create lots of shard
+rebalancing as cluster members restart.
+
+We present two restart procedures: rolling-restart and full-restart. Where a rolling
```

Comment:
s/We present/This section describes

Do you think it would help to link to the sections?